__tests__/endpoints.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest');
+const seed = require('../db/seeds/seed.js');
+const { topicData, userData, articleData, commentData } = require('../db/data/test-data/index.js'); 
+const db = require('../db/connection.js');
+const app = require('../app.js');
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+    test('returns status 200 and responds with an array of topic objects containing "slug" and "description" properties', () => {
+        return request(app).get('/api/topics')
+            .expect(200)
+            .then((res) => {
+                const topics = res.body.topics;
+                expect(topics).toHaveLength(3)
+                topics.forEach((topic) => {
+                    expect(topic).toHaveProperty('slug');
+                    expect(topic).toHaveProperty('description');
+                })
+            })
+    })
```

Comment:
Well done on this test!

Great that you've ensured the properties exist on the response. Keep in mind however that you could make this an even stronger test by also checking the type of the value that is returned.

There's a few ways we can do this - toHaveProperty can actually accept a second argument for the expected value like so:
          expect(article).toHaveProperty("author", expect.any(String));
          
OR, you can use the jest matcher .toMatchObject for a similar purpose:

expect(article).toMatchObject({
author : expect.any(String)
<... other properties>
})

Or you may choose to test that the typeof a property is as you expect. (Which also implicitly tests that they exist)